zerver/openapi/zulip.yaml

```diff
@@ -726,37 +726,37 @@ paths:
                                   type: array
                                   description: |
                                     A list of dictionaries, where each dictionary contains
-                                    information about one of the newly unsubscribed streams.
+                                    information about one of the newly unsubscribed channels.
                                   items:
                                     type: object
                                     additionalProperties: false
                                     description: |
-                                      Dictionary containing details about the unsubscribed stream.
+                                      Dictionary containing details about the unsubscribed channel.
                                     properties:
                                       stream_id:
                                         type: integer
                                         description: |
-                                          The ID of the stream.
+                                          The ID of the channel.
                                       name:
                                         type: string
                                         description: |
-                                          The name of the stream.
+                                          The name of the channel.
                               additionalProperties: false
                               example:
                                 {
                                   "type": "subscription",
                                   "op": "remove",
                                   "subscriptions":
-                                    [{"name": "test_stream", "stream_id": 9}],
+                                    [{"name": "test", "stream_id": 9}],
                                   "id": 0,
                                 }
                             - type: object
                               description: |
                                 Event sent to a user's clients when a property of the user's
-                                subscription to a stream has been updated. This event is used
+                                subscription to a channel has been updated. This event is used
                                 only for personal properties like `is_muted` or `pin_to_top`.
-                                See the [stream update event](/api/get-events#stream-update)
-                                for updates to global properties of a stream.
+                                See the [`stream op: update` event](/api/get-events#stream-update)
```

Comment:
Is the space after `op: ` intentional? Anyway I think the colloquial "channel update event" feels better than trying to encode the event type in that format, given that we've got a link at least.